applications/serial_lte_modem/src/nrfcloud/slm_at_nrfcloud.c

```diff
@@ -418,20 +418,6 @@ static bool handle_cloud_cmd(const char *buf_in)
 			k_work_submit_to_queue(&slm_work_q, &cloud_cmd);
 			ret = true;
 		}
-	/* Format expected from nrf cloud:
```

Comment:
Removing this alters our functionality a bit, if this data still keeps coming. We would now start sending the disconnect event as raw data in #XNRFCLOUD: notifications.

If this is still coming through NRF_CLOUD_EVT_RX_DATA_GENERAL, then this part of code should be kept. But if NRF_CLOUD_EVT_RX_DATA_GENERAL no longer sends this, then this OK to be removed.

Reply:
You will only get one event for this -- `NRF_CLOUD_EVT_RX_DATA_DISCON`. You will only get `NRF_CLOUD_EVT_RX_DATA_GENERAL` if it is not a DISCON message.